e2e/init/init_test.go

```diff
@@ -83,7 +83,7 @@ var _ = Describe("init flow", func() {
 			Expect(len(app.Routes)).To(Equal(1))
 			Expect(app.Routes[0].Environment).To(Equal("test"))
 			Expect(app.Routes[0].Path).To(Equal(appName))
-			resp, fetchErr := http.Get(fmt.Sprintf("http://%s/%s/", app.Routes[0].URL, app.Routes[0].Path))
+			resp, fetchErr := http.Get(fmt.Sprintf("http://%s/", app.Routes[0].URL))
```

Comment:
I believe the other e2e tests "multi-app-project" and "multi-env-project" need to be updated similarly